.github/workflows/quarkus.yml

```diff
@@ -0,0 +1,348 @@
+name: Quarkus tests
+
+on: [push]
+
+env:
+  # Workaround testsuite locale issue
+  LANG: en_US.UTF-8
+  DB_USER: hibernate_orm_test
+  DB_PASSWORD: hibernate_orm_test
+  DB_NAME: hibernate_orm_test
+  NATIVE_TEST_MAVEN_OPTS: "-B -Dtest-postgresql -Dtest-elasticsearch -Dtest-keycloak -Dtest-dynamodb -Dtest-mysql -Dtest-mariadb -Dmariadb.url='jdbc:mariadb://localhost:3308/hibernate_orm_test'  -Dtest-mssql -Dtest-vault -Dtest-neo4j -Dnative-image.xmx=5g -Dnative -Dformat.skip install"
+  MX_GIT_CACHE: refcache
+  JAVA_HOME: ${{ github.workspace }}/openjdk
+
+jobs:
+  build-mandrel:
+    name: Mandrel build
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+        path: mandrel
```

Comment:
Isn't branch info missing here? Otherwise I think it just checks out master.

Reply:
By default it checks out the branch that triggered the build, in this case it will be 20.1 (or any private branch of it we create for development).

Reply:
You can verify this by comparing the tests results from 20.1 with those of master (master currently fails with OpenJDK 11, while 20.1 does not)